hardware/nvidia/platform/t19x/jakku/kernel-dts/tegra194-siemens-ipc520a.dts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2024, Zededa GmbH. All rights reserved.
```

Comment:
I think we use Zededa, Inc for work for hire.

Reply:
ok, will fix it :+1: